man/systemd.network.xml

```diff
@@ -208,6 +208,20 @@
           integer in the range 0—4294967294. Defaults to unset.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>TransmitQueues=</varname></term>
+        <listitem>
+          <para>Specifies the devices's number of transmit queues. An unsigned
+          integer in the range 1—4096. When unset, the kernel's default will be used.</para>
```

Comment:
we usually spell out ranges in the docs as `1…4096` rather than `1-4096`. Please do so here too